main/options/options.cc

```diff
@@ -422,9 +422,10 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                     "Modules that should never be collapsed into their parent. This helps break cycles "
                                     "in certain cases. (e.g. Foo::Bar::Baz)",
                                     cxxopts::value<vector<string>>());
-    options.add_options("advanced")("autogen-autoloader-pbal-namespaces",
-                                    "Namespaces for which path-based autoloading is enabled.",
-                                    cxxopts::value<vector<string>>());
+    options.add_options("advanced")("autogen-autoloader-pbal-non-annotated",
+                                    "When this flag is set, packages that don't have 'autoloader_compatibility' "
+                                    "annotations use path-based autoloading",
+                                    cxxopts::value<bool>());
```

Comment:
Why do we need this flag? Are there going to be some cases where we pass this flag and other cases where we don't?

Or are we just going to always pass this flag in all Sorbet invocations?

Reply:
Right now the flag is there because I want to be able to bump Sorbet without having this functionality on by default. It will be removed very soon.